src/styles.scss

```diff
@@ -1292,6 +1292,11 @@ mat-icon.star {
   text-overflow: ellipsis;
 }
 
+.notification-display p {
+  text-align: center;
+  font-size: 18px;
```

Comment:
When we were looking at this, wasn't there an issue with margin-bottom? Do you need to add something for that?